tests/rules/flowbits/test.yaml

```diff
@@ -0,0 +1,100 @@
+requires:
+    min-version: 7.0.0
```

Comment:
For now, let's keep `min-version: 8.0`, as we're not sure if we will bring this work to suricata 7, yet :)

Reply:
got it.